python/mxnet/runtime.py

```diff
@@ -77,11 +77,9 @@ class Features(collections.OrderedDict):
     def __new__(cls):
         if cls.instance is None:
             cls.instance = super(Features, cls).__new__(cls)
+            super(Features, cls.instance).__init__([(f.name, f) for f in feature_list()])
         return cls.instance
 
-    def __init__(self):
```

Comment:
can you leave __init__ with "pass" or needs to be removed?

Reply:
Since we are reusing the instance, are you sure we are calling ctor all the time?

Reply:
discussed offline. resolving!